src/index.html

```diff
@@ -10,23 +10,26 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="search-bar">
+      <input
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+        class="search-bar__big input"
```

Comment:
Remember that class names should represent the meaning of the content, not the styles or tag names. In this case, the class name 'input' doesn't seem to describe the content of the element. Consider renaming it to something more descriptive.